Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -431,25 +443,33 @@ class WebRTCClient: NSObject {
         await assignTracksToParticipants()
         let state = await self.state.connectionState
         if state == .connected {
-            await updateParticipantsSubscriptions()
+            try? await updateParticipantsSubscriptions()
```

Comment:
Why not `try`?

Reply:
because we still want to update the local state (the next lines) even if the request failed